<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgInitUserMap</class>
 <widget class="QDialog" name="DlgInitUserMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="lblMapName">
     <property name="text">
      <string>Map name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMapSize">
     <property name="text">
      <string>Size (X, Y)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lnMapSizeX"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lnMapSizeY"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnConfirm">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Map creation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lnMapName"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
